k4FWCore/components/PodioOutput.cpp

```diff
@@ -38,6 +42,20 @@ StatusCode PodioOutput::initialize() {
     return StatusCode::FAILURE;
   }
 
+  // check whether output directory needs to be created and eventually create it
+  auto outDirPath = std::filesystem::path(m_filename.value()).parent_path();
+  if (!outDirPath.empty() && !std::filesystem::is_directory(outDirPath)) {
+    std::error_code ec;
+    std::filesystem::create_directories(outDirPath, ec);
```

Comment:
I think we could check the return value here in addition to the error code to also give a debug message if the directory was already there and was not created due to that.

Reply:
But the the `std::filesystem::create_directories()` will be called only if the directory does not exists (line47).